mmv1/third_party/terraform/utils/utils.go

```diff
@@ -527,26 +527,26 @@ func fake404(reasonResourceType, resourceName string) *googleapi.Error {
 	}
 }
 
-// validate name of the gcs bucket.
+// validate name of the gcs bucket. Guidelines are located at https://cloud.google.com/storage/docs/naming-buckets
+// this does not attempt to check for IP addresses or close misspellings of "google"
 func checkGCSName(name string) error {
-	MAX_LENGTH := 63
-	var err error
+	if strings.HasPrefix(name, "goog") {
+		return fmt.Errorf("error: bucket name %s cannot start with %q", name, "goog")
+	}
+
+	if strings.Contains(name, "google") {
+		return fmt.Errorf("error: bucket name %s cannot contain %q", name, "google")
+	}
+
+	valid, _ := regexp.MatchString("^[a-z0-9][a-z0-9_.-]{1,220}[a-z0-9]$", name)
+	if !valid {
+		return fmt.Errorf("error: bucket name validation failed %v", name)
```

Comment:
```suggestion
		return fmt.Errorf("error: bucket name validation failed %v. See https://cloud.google.com/storage/docs/naming-buckets", name)
```

would a link to the documentation be helpful?

Reply:
@daniel-cit not a bad idea